runner.go

```diff
@@ -121,7 +121,10 @@ func (r *runner) outputOnStop() {
 }
 
 func (r *runner) spawnWorkers(spawnCount int, quit chan bool, spawnCompleteFunc func()) {
-	log.Println("Spawning", spawnCount, "clients immediately")
+	for i := 0; i > spawnCount; i-- {
```

Comment:
OK, so u think it might be a better way to define another function such as stopWorkers which could handle spawnCount < 0

Reply:
> What about a new stopWorkers function?

OK, so u think it might be a better way to define another function such as stopWorkers which could handle spawnCount < 0